code/modules/research/designs/cell.dm

```diff
@@ -89,6 +89,13 @@
 	name = "Soteria \"Atomcell 50S\""
 	build_path = /obj/item/cell/small/moebius/pda
 
+//Magic rock cell
+
+/datum/design/research/item/powercell/large/ameridian
+	name = "Soteria \"Ameridian 20000L\""
+	build_path = /obj/item/cell/large/ameridian
+	materials = list(MATERIAL_AMERIDIAN = 5)
```

Comment:
This is the ameridian shards. We can make it use reagents like Liquid Ameridian instead, I think it would be better.
Also IIRC this mean that it *only* require 5 ameridian shards to make, way cheaper overall than the omega cell.